checksets/03-regular_commit_proposal_list.dhall

```diff
@@ -131,9 +131,17 @@ let checks =
               "It contains multiple GroupContextExtension proposals."
               "section-12.2-3.8"
           )
-          types.Status.Unknown
-          types.CodeRefs/empty
-          types.CodeRefs/empty
+          types.Status.Complete
+          ( types.CodeRefs/new
+              [ types.CodeRef/new
+                  "openmls::group::public_group::validate_group_context_extensions_proposal"
```

Comment:
Should we make these link into the code to be a little more useful?

Reply:
The reason I went with these links was that they don't get stale very quickly (compared to links to line ranges in snapshots of files), but until we have tooling to deal with module paths, I agree it would be nice to have more pragmatic links as well.

I will open an issue to let `CodeRef`s also have a plain permalink to the code in question.